packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -193,9 +193,17 @@ export default class BackupNg {
 
           const remotes = {}
           const xapis = {}
+          const remoteErrors = {}
           await waitAll([
             asyncMapSettled(remoteIds, async id => {
-              const remote = await app.getRemoteWithCredentials(id)
+              let remote
+              try {
+                remote = await app.getRemoteWithCredentials(id)
+              } catch (error) {
+                log.warn('Error while instantiating remote', { error, remoteId: id })
```

Comment:
Making it appear in the task would be better but I don't know how to do it easily.

Reply:
I didn't find any easy way without a notable refactoring that would delay the release